### PR TITLE
Added support for tracing_port

### PR DESCRIPTION
**Pull request by howardyoo**
proxy client needs tracing_port to be specified, if exists

### CODE REVIEW

File: wavefront_pyformance/wavefront_reporter.py
Comment (haosong) [inline wavefront_pyformance/wavefront_reporter.py:121]: The reason we don't expose tracing port is that `pyformance` is basically a metrics library, which would not be reporting tracing data. But you can still create the wavefront client yourself and pass it as `wavefront_client` manually.
Any use case you want to have the tracing port directly here?

Comment (howardyoo) [reply wavefront_pyformance/wavefront_reporter.py:121]: Yes, I'm currently working on a HOL on polyglot application (beachshirts). Due to the nature of HOL having a restricted environment, I had to use proxy mode to use http proxy, which would make the work a whole lot easier. Then, I found out that the application was not running, and found out the warehouse service (which was made and run using python) was not able to receive the tracing_port from the reporter. I had to add the tracing_port as an init input to finally make it work.

**Review by haosong — APPROVED** [review]

**Comment by ustinov** [comment]
@howardyoo please make sure you bump the package version before merging this

**Comment by howardyoo** [comment]
Okay.. which package version are you referring to? Not sure what to update...

**Comment by ustinov** [comment]
> Okay.. which package version are you referring to? Not sure what to update...

This one here https://github.com/wavefrontHQ/wavefront-pyformance/blob/master/setup.py#L24

**Comment by howardyoo** [comment]
Ok... updated the version to 1.0.1

**Comment by ustinov** [comment]
> Ok... updated the version to 1.0.1

Version `1.0.1` is current one [*0], so it should be `1.0.2` for you.
The reason why we need an unique version every time is that we deploy every merged commit to PyPI [*1] automatically, and it doesn't allow overwriting existing uploads.

[*0] https://github.com/wavefrontHQ/wavefront-pyformance/blob/master/setup.py#L24
[*1] https://test.pypi.org/project/wavefront-pyformance/#history

**Comment by howardyoo** [comment]
Ok, changed the version to 1.0.2

**Comment by ustinov** [comment]
We'll have to wait for @sushantdewan123 or @vikramraman to merge this

**Pull request closed without merging**
